app/src/main/res/values/strings.xml

```diff
@@ -3,43 +3,34 @@
 
     <string name="loginTitle">forzenbook</string>
```

Comment:
Strings resources are snake case, see "app_name" that was generated for you

Reply:
right click -> refactor -> rename -> carefully edit -> enter -> wait -> repeat

fixed this one up :3